src/styles/_catalog.scss

```
@@ -1,22 +1,31 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: 400px;
+  grid-auto-rows: 300px;
+  place-content: center;
+  grid-gap: 48px;
+  justify-content: center;
+  justify-items: center;
```

if i will delete it - nothing will be changed
```suggestion
  grid-gap: 48px;
  justify-content: center;
```